src/v1/function-builder.ts

```diff
@@ -192,6 +192,34 @@ function assertRuntimeOptionsValid(runtimeOptions: RuntimeOptions): boolean {
       );
     }
   }
+
+  if (
+    runtimeOptions.invoker !== undefined &&
```

Comment:
I think this is a bit of a redundant check - if runtimeOptions.invoker is undefined, then typeof runtimeOptions will equal undefined as well.

I think you can just get rid of this undefined check.

Reply:
Removed